db/index.js

```diff
@@ -1,131 +1,130 @@
+          if (err) {
+            console.log(err);
+            return;
+          }
+        });
+      }
+    });
   });
 };
 
-const findId = (inputGame, cb) => {
-  Review.find({ id: inputGame }).exec((err, results) => {
+// read
+const findId = (id, cb) => {
+  client.execute('SELECT * FROM reviews WHERE gameid = ?', [id], { prepare: true }, (err, results) => {
     if (err) {
-      cb(err);
+      console.log(err);
+      return;
     }
-    cb(null, results);
+    const addLineBreaks = (str) => str.replace(/Þ/g, '\n');
+    const data = [];
+    results.rows.forEach((item) => {
+      const newRow = {
+        id: item.get('gameid'),
+        game: item.get('game'),
+        game_reviews: item.get('game_reviews'),
+        rating: item.get('rating'),
+        hours: item.get('hours'),
+        description: addLineBreaks(item.get('description')),
+        helpful: item.get('helpful'),
+        funny: item.get('funny'),
+        date_posted: item.get('date_posted'),
+        language: 'EN',
+        thread_length: item.get('thread_length'),
+        user: {
+          id: item.get('user_id'),
+          username: item.get('user_username'),
+          steam_purchaser: item.get('user_steam_purchaser'),
+          recommended: item.get('user_recommended'),
+          numProducts: item.get('user_numproducts'),
+          numReviews: item.get('user_numreviews'),
+          icon: item.get('user_icon'),
+          player_type: item.get('user_player_type'),
+          xp: item.get('user_xp'),
+          friend_level: item.get('user_friend_level'),
+          steam_level: item.get('user_steam_level'),
+        },
+      };
+      data.push(newRow);
+    });
+    cb(null, data);
   });
 };
 
-const updateId = (id, changes, cb) => {
-  Review.update({ id }, changes, (err, results) => {
+// update
+const updateId = (gameid, uid, field, value, cb) => {
+  let query;
+  if (field === 'helpful') {
+    query = 'UPDATE reviews SET helpful = ? WHERE gameid=? AND user_id=?';
+  } else {
+    query = 'UPDATE reviews SET funny = ? WHERE gameid=? AND user_id=?';
+  }
+
+  client.execute(query, [value, gameid, uid], { prepare: true }, (err, results) => {
     if (err) {
-      cb(err);
+      console.log(err);
+      return;
     }
+    console.log(results);
     cb(null, results);
   });
 };
 
-const deleteId = (id, cb) => {
-  Review.deleteMany({ id }, (err) => {
+// delete review from specified user for given gameid
+const deleteId = (gameid, uid, cb) => {
+  client.execute('DELETE FROM reviews WHERE gameid=? AND user_id=?', [gameid, uid], { prepare: true }, (err, results) => {
     if (err) {
       console.log(err);
+      return;
     }
-    cb();
+    console.log(results);
+    cb(null, results);
   });
 };
 
-
 module.exports.create = create;
 module.exports.findId = findId;
 module.exports.updateId = updateId;
 module.exports.deleteId = deleteId;
```

Comment:
A cool shortcut for this would be:
`module.exports = { create, findId, updateId, deleteId };`

Reply:
i agree.  its done :)